apps/astarte_realm_management_api/lib/astarte_realm_management_api_web/controllers/interface_controller.ex

```diff
@@ -24,6 +24,16 @@ defmodule Astarte.RealmManagement.APIWeb.InterfaceController do
 
   action_fallback Astarte.RealmManagement.APIWeb.FallbackController
 
+  def index(conn, %{"realm_name" => realm_name, "detailed" => "true"}) do
+    with {:ok, interfaces} <-
+           Astarte.RealmManagement.API.Interfaces.list_interfaces(realm_name, %{
```

Comment:
Nit: add an `alias Astarte.RealmManagement.API.Interfaces` above, so that this call is on one line only (hopefully)

Reply:
Alias already added. Just removed full path now.